etc/adminhtml/system.xml

```diff
@@ -93,6 +93,12 @@
                 <group id="additional" translate="label" type="text" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Additional options</label>
                     <attribute type="expanded">1</attribute>
+                    <field id="pdp_frequency_selector" translate="label" type="select" sortOrder="80" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
please add default value to "config.xml"

Reply:
Added